### PR TITLE
Relationship model inherit HoustonModel

### PR DESCRIPTION
**Pull request by colinwkingen**
Tiny change to make Relationship model inherit HoustonModel and remove a redundant method.

### CODE REVIEW

**Review by naknomum — APPROVED** [review]
looks good.

File: app/modules/relationships/models.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def delete(self):
         relationship.delete()
 
 
-class Relationship(db.Model, Timestamp):
+class Relationship(db.Model, HoustonModel, Timestamp):
```
Comment (hwindsor) [inline app/modules/relationships/models.py:46]: HoustonModel inherits FeatherModel inherits TimestampViewed inherits Timestamp so Timestams shouldn't be needed anymore

**Pull request merged**